### PR TITLE
Adapt 1909  horizontal small circle card

### PR DESCRIPTION
**Pull request by katrialesser**
# READY FOR REVIEW

# Summary
- Add basic horizontal card styles and styles for small circle horizontal card

# Review By (Date)
- 3/24/21 morning if possible, before sprint closes at lunchtime. (Would like to close the ticket)

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out the test site: https://saauserguide-test.sites.stanford.edu/all-card-test-examples
2. Look at the horizontal cards with small circles
3. Compare with styles on this page: https://saauserguide.sites.stanford.edu/site-building-guide/saa-components-stanford-sites/cards/small-horizontal-circle-card 
4. You can look at the figma design here: https://www.figma.com/file/TXNBECAGcEkZ3vwc6xcZ3g/SAA-Design-System?node-id=192%3A628 if you aren't sure about some spacing settings.

## Front End Validation
- [X ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [X] Design is approved by @ user? Design previously approved by Kerri; design is already in use, new implementation

### Code
- [X ] Are the naming conventions following our standards?
- [ X] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s) ADAPT-1715 & ADAPT-1909
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Comment by jenbreese** [comment]
@katrialesser, the card font size and leading is different from the text field faux cards.  I  noticed the `.su-card .su-card__contents p` is set to `20px`.  I think they should be em. The circle image needs to be a square for the circle to look correct. I'm not sure if that is expected or if we want the theme to handle that.
image


Was the font size and leading a design change? Why not change this line to https://github.com/SU-SWS/saa_victoria_subtheme/blob/d02c3c9622777d554101f213479503dab0159452/src/scss/components/card/_card.scss#L14 `@include big-paragraph;` ?

**Comment by katrialesser** [comment]
> @katrialesser, the card font size and leading is different from the text field faux cards. I noticed the `.su-card .su-card__contents p` is set to `20px`. I think they should be em. The circle image needs to be a square for the circle to look correct. I'm not sure if that is expected or if we want the theme to handle that.
> image
> 
> Was the font size and leading a design change? Why not change this line to
> 
> https://github.com/SU-SWS/saa_victoria_subtheme/blob/d02c3c9622777d554101f213479503dab0159452/src/scss/components/card/_card.scss#L14
> 
> `@include big-paragraph;` ?

1 - I think I'm a little confused about the line height difference or where to see that. When I inspect a card on the saauserguide site I see line-height: 1.4em, the same as I see on my branch and in this card.scss file   -   https://saauserguide.sites.stanford.edu/site-building-guide/saa-components-stanford-sites/cards/icon-cards

If you can link more clearly where you see line height differences between the faux cards and the new cards please share a link so I can inspect the line-height difference you're seeing. 

I do not know if they have changed figma since we have implemented or if it has somehow diverged.. it is possible. So I think we should go off of **implementing exactly what is already implemented in the faux cards online** because they have visually approved that with stakeholders etc.

2 - I do not believe that big-paragraph mixin will follow the specs; the specs want 18px font size for smaller sizes and then 20px font size medium and up. If you inspect and change it to 2.1rem you can see the font size changes and they specifically wanted these sizes so we need to keep it the same. They are only using 2 font styles slightly different from our modular typography and everything else follows it so this isn't a problem.

3 - if we change all our font settings that use px to rem that is a whole different ticket of refactoring the font sizes. Not part of our work right now. The specs were given specifically to use those px sizes at the breakpoints specified, which is technically different than using modular spacing and rem's and having it adjust with 3 different base px sizes set in decanter, so that is a CHANGE in font sizes. Which we aren't doing at this time and not doing until it is requested or approved.

**Comment by jenbreese** [comment]
Notes from quick zoom with @katrialesser 

Move line height: 1.4  from the general card file to the icon only.
Client should see no visual difference.

Confirmed: client must upload 1:1 image.

Thanks!

File: src/scss/components/card/horizontal-card/_horizontal-card-general.scss [hunk 1]
```diff
@@ -2,9 +2,29 @@
...
   .su-card {
-     // placeholder
+    @apply flex;
+    & > .image {
+      @apply hidden;
+
+      @include grid-media-min('sm') {
+        @apply
+        block
+        mt-0;
```
Comment (katrialesser) [inline src/scss/components/card/horizontal-card/_horizontal-card-general.scss:12]: @jenbreese also note that this is our standard format when we are @apply-ing multiple lines in a row.

**Review by jenbreese — APPROVED** [review]
Looks good and what we discussed.

**Pull request merged**